packages/angular-material/src/controls/date.renderer.ts

```diff
@@ -0,0 +1,38 @@
+import { Component } from '@angular/core';
+import { NgRedux } from '@angular-redux/store';
+import { isDateControl, JsonFormsState, RankedTester, rankWith } from '@jsonforms/core';
+import { JsonFormsControl } from '@jsonforms/angular';
+
+@Component({
+  selector: 'DateControlRenderer',
+  template: `
+  <mat-form-field fxFlex>
+    <mat-label>{{ label }}</mat-label>
+    <input
+        matInput
+        (dateChange)="onChange($event)"
+        [value]="data"
+        placeholder="{{ description }}"
+        [id]="id"
+        [formControl]="form"
+        [matDatepicker]="myDatepicker"
+    >
+    <mat-datepicker-toggle matSuffix [for]="myDatepicker"></mat-datepicker-toggle>
+    <mat-datepicker #myDatepicker></mat-datepicker>
```

Comment:
I'd change the ref name

Reply:
do you whether the ref name must be unique across components, or only unique inside a component?

Reply:
As I understand this article https://blog.angular-university.io/angular-viewchild/#visibilityscopeofviewchildtemplatequeries correctly, then the view ref is bound to the component scope, and thus this should be fine